webpack.config.js

```diff
@@ -11,7 +11,7 @@ const config = {
     entry: './src/extension.ts', // the entry point of this extension, 📖 -> https://webpack.js.org/configuration/entry-context/
     output: {
         // the bundle is stored in the 'dist' folder (check package.json), 📖 -> https://webpack.js.org/configuration/output/
-        path: path.resolve(__dirname, 'dist'),
+        path: path.resolve(__dirname, 'out', 'src'),

```

Comment:
what is the purpose of the folder change?

Reply:
because the entrance has been set by `main` property in `package.json`

Reply:
ok, why change in `main` in `package.json` then?

Reply:
if we don't bundle, the ts files will be compiled to /out/src/**.js

Reply:
I see, thanks